src/cmd/team.go

```diff
@@ -12,17 +12,19 @@ import (
 )
 
 var exampleTeamCmd = &cobra.Command{
-	Use:   "team",
-	Short: "Example team",
-	Long:  `Example team`,
+	Use:     "team",
+	Aliases: common.GetAliases("Team"),
+	Short:   "Example team",
+	Long:    `Example team`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println(getExample[opslevel.TeamCreateInput]())
 	},
 }
 
 var createTeamCmd = &cobra.Command{
-	Use:   "team NAME",
-	Short: "Create a team",
+	Use:     "team NAME",
+	Aliases: common.GetAliases("Team"),
```

Comment:
I've just started looking at these. The help docs for this (`opslevel create team --help`) show me:
```bash
Usage:
  opslevel create team NAME [flags]

Aliases:
  team, team, teams
```
Which is actually pretty confusing.
1. `team` is the default already and listed twice under `Aliases:`
2. `teams` suggests that I can create multiple teams with this one command. But the usage says `Short: "Create a team"`

Reply:
@davidbloss this has been fixed.

1. Aliases are not shown unnecessarily in the help menu anymore
2. Example, create, update, delete, assign, unassign don't support plural nouns because it can cause confusion
3. Only list, import support plural nouns

Create team (no aliases) looks normal again:

```
$ op create team -h
Create a team

Usage:
  opslevel create team NAME [flags]
```

Here's create system (has alias) and list system:

```
$ op create system -h
Create a system

Usage:
  opslevel create system [flags]

Aliases:
  system, sys

$ op list system -h
Lists the systems

Usage:
  opslevel list system [flags]

Aliases:
  system, systems, sys
```